app-virtualization/qemu/34-guest-agent/overrides/usr/lib/systemd/system/qemu-guest-agent.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=QEMU Guest Agent
+BindsTo=dev-virtio\x2dports-org.qemu.guest_agent.0.device
+After=dev-virtio\x2dports-org.qemu.guest_agent.0.device
+
+[Service]
+Type=simple
+ExecStart=-/usr/bin/qemu-ga
+StandardError=journal
+Restart=always
+RestartSec=0
+
+[Install]
```

Comment:
Why is this section empty?

Reply:
The service will be triggered by UDev rule `/usr/lib/udev/rules.d/99-qemu-guest-agent.rules`